management/server/http/peers_handler.go

```diff
@@ -228,6 +228,20 @@ func (h *PeersHandler) GetAccessiblePeers(w http.ResponseWriter, r *http.Request
 		return
 	}
 
+	if !user.HasAdminPower() && !user.IsServiceUser {
+		peer, err := h.accountManager.GetPeer(r.Context(), account.Id, peerID, user.Id)
```

Comment:
We are introducing an unnecessary get account here. I suggest either optimizing the GetPeer or reusing the account in line 218, but consider PR #2615